bridges/evm/contracts/example/NeoTCD.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity 0.5.14;
+pragma experimental ABIEncoderV2;
+
+interface Bridge {
```

Comment:
Import this interface from `IBridge.sol` like other example?

Reply:
done